scripts.js

```diff
@@ -1,109 +1,115 @@
-//       ***********************
-//            INSTRUCTIONS
-//       ***********************
+let currentMarker = "X";
+let board = [
+  ["", "", ""],
+  ["", "", ""],
+  ["", "", ""],
+];
 
-// 1. Read the code below one block at a time.
-// 2. Look for the @TODOs, and figure out how to fix them.
-    // next to each @TODO you will find tasks that need to be finished
-
-// The variable will change from X to O based on what player turn it is. We need to hold this so we can place an X or O on the board when they're clicked.
-let currentMarker = 'X'
-
-
-
-
-// this "handleClick" function is called when a box is clicked. Here, "element" will hold the same value as "this" does in the HTML. 
-// "this" is a special word in JS but "element" could have been "thing" or "el" or whatever we wanted it to be as long as we use it again in the "console.log" statement
 const handleClick = (element) => {
+  console.log(`The element you clicked on has an id:  ${element.id}`);
 
-  // this uses the "log" method on the "console" to log out the element's id so we can see it with our human eyes
-  console.log(`The element you clicked on has an id:  ${element.id}`)
-
-  // this next line prevents an X being changed to an O or an O being changed to an X by...
-  //  checking to see if the square clicked has anything in it, if not continue
-  if(!document.getElementById(element.id).innerHTML){
-    addMarker(element.id)
+  if (!document.getElementById(element.id).innerHTML) {
+    addMarker(element.id);
   }
-}
-
-
-
-
-
+};
 
-
-
-
-
-
-// this function places the "currentMarker" inside the HTML element that was clicked and calls the "changeMarker" function.
 const addMarker = (id) => {
+  console.log(`*** The current marker is:  ${currentMarker}. ***`);
+  console.log(
+    `Therefore, a  "${currentMarker}"  should be placed in the square with the id:  ${id}`
+  );
 
-  // @TODO-1: Open the console tab in your Chrome Inspector Tool and click on the top-left square to see what's logged to the console. 
-  console.log(`*** The current marker is:  ${currentMarker}. ***`)
-  console.log(`Therefore, a  "${currentMarker}"  should be placed in the square with the id:  ${id}`)
-  
-  // @TODO-2: Build a line of code that will set the innerHTML property of the element that was clicked to the "currentMarker"
-  
-  // @TODO-2.5: MIX & MATCH, You will need the following pieces of code to build that line:
-  // = currentMarker
-  // .getElementById(id)
-  // document
-  // .innerHTML 
-
-  changeMarker( document.getElementById(id).innerHTML = currentMarker)
-
-
-}  
-
-
-
+  const row = parseInt(id.charAt(0));
+  const column = parseInt(id.charAt(2));
+  board[row][column] = currentMarker;
 
+  // changeMarker(document.getElementById(id).innerHTML = currentMarker)
 
+  checkForWin((document.getElementById(id).innerHTML = currentMarker));
```

Comment:
you're passing something in here. but what you're actually doing is setting currentMarker onto the specific selected square. and you need that part. but not in the parentheses for the checkwin function. i'd pull it out and put it right on top of checkforWin()

Reply:
```
document.getElementById(id).innerHTML = currentMarker;
checkForWin();
```